src/main/java/com/starkinfra/PixInfraction.java

```diff
@@ -134,9 +150,12 @@ public PixInfraction(Map<String, Object> data) throws Exception {
 
         this.referenceId = (String) dataCopy.remove("referenceId");
         this.type = (String) dataCopy.remove("type");
+        this.method = (String) dataCopy.remove("method");
         this.description = (String) dataCopy.remove("description");
         this.tags = (String[]) dataCopy.remove("tags");
+        this.fraudType = (String) dataCopy.remove("fraudType");
         this.creditedBankCode = (String) dataCopy.remove("creditedBankCode");
```

Comment:
Por que o creditedBankCode não é `this.creditedBankCode = null`, tendo em vista que é um atributo e não um parâmetro?

Reply:
Pq não fui eu que fiz :D